openfisca_france/parameters/prestations/majconj_plaf/plafond_des_ressources_personnelles_du_conjoint_pour_beneficier_de_la_majoration_pour_personne_a_charge.yaml

```
@@ -169,3 +169,5 @@ values:
     value: 8990.2
   2016-04-01:
     value: 8999.8
+  2021-04-01:
+    value: 10271.95
```

À défaut d'un lien plus stable : 
```suggestion
    value: 10271.95
    reference: https://www.legislation.cnav.fr/Documents/circulaire_cnav_2021_01_11012021.pdf
```
> Aussi cité dans les [barèmes cnav](https://www.legislation.cnav.fr/Pages/bareme.aspx?Nom=derniere_valeur_derniere_valeur_bar)